core/lexicon/en/default.inc.php

```diff
@@ -314,7 +315,7 @@
 $_lang['orm_container_rename'] = 'Rename Container';
 $_lang['orm_container_remove'] = 'Delete Container';
 $_lang['orm_container_remove_confirm'] = 'Are you sure you want to delete this container and all attributes below it? This is irreversible.';
-$_lang['pagetitle'] = 'Resource\'s title';
+$_lang['pagetitle'] = 'Resource‘s Title';
```

Comment:
I believe this should stay a `'` and not a `‘`.

Reply:
Ok, I'll change this ... but I'd love to see us displaying proper typographical quotes _somehow_ ;-)